lib/db/feature-toggle-store.js

```diff
@@ -96,6 +98,9 @@ class FeatureToggleStore {
             enabled: data.enabled ? 1 : 0,
             archived: data.archived ? 1 : 0,
             strategies: JSON.stringify(data.strategies),
+            variations: data.variations
+                ? JSON.stringify(data.variations)
+                : '[]',
```

Comment:
I would prefer to have the database migrate script (which is missing from this changeset) to default this column to an empty array.